pkg/bundles/map.go

```diff
@@ -107,6 +107,28 @@ var (
 	remoteWrite      = remote.Write
 )
 
+// TestWithFakes temporarily switches the remote writes with fakes for testing
+// ideally we'd have a nicer way to do this with a fake container registry host or something
+func TestWithFakes(fn func() error) error {
+	oldWriteIndex := remoteWriteIndex
+	oldWrite := remoteWrite
```

Comment:
Do you want to update the unit tests here to use this as well?  That way it's less likely I'll inadvertently break this 😅

Reply:
yeah ;) - wanted to double check this change wasn't too awful to include first to keep PRs smallish. I can easily roll this PR into the unit test PR once the clients/namespace/ctx stuff is sorted